resources/mysqlbinlog/mysqlbinlog_8.0.28_linux_glibc_2.17_x84_64_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build linux && amd64
```

Comment:
Can we follow the pattern of resources/postgres/postgres.go so that we only need to have a single test for different platforms?

Reply:
SG.